app/views/spree/orders/_form.html.haml

```diff
@@ -35,13 +35,13 @@
           %td.text-right
             %span.order-total.item-total= display_checkout_subtotal(@order)
           %td
-
-        %tr
-          %td.text-right{colspan:"3"}
-            = t :orders_form_admin
-          %td.text-right
-            %span.order-total.distribution-total= display_checkout_admin_and_handling_adjustments_total_for(@order)
-          %td
+        -if display_checkout_admin_and_handling_adjustments_total_for(@order)
```

Comment:
Did you consider moving the conditional in `app/helpers/checkout_helper.rb:29` to here as:

```ruby
if display_checkout_admin_and_handling_adjustments_total_for(@order) == Spree::Money.new(0 , currency: @order.currency)
```

This way `display_checkout_admin_and_handling_adjustments_total_for` will keep returning just a data type, which keeps thing simple and reusable.

Reply:
Okay thanks for pointing that out. I will make that change.